gravitee-apim-portal-webui-next/src/components/card/card.component.html

```diff
@@ -0,0 +1,34 @@
+<!--
+
+    Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+    
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    
+            http://www.apache.org/licenses/LICENSE-2.0
+    
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<mat-card class="api-card">
+  <mat-card-content>
+    <div class="api-card__header">
+      <img i18n-alt="@@logo" ngSrc="assets/images/logo.png" height="40" width="40" alt="Company logo" class="api-card__header-logo" />
+      <div class="api-card__header-content">
+        <div class="api-card__header-content-title">{{ title }}</div>
+        <div class="api-card__header-content-version">{{ version }}</div>
+      </div>
+    </div>
+    <div class="api-card__content">
+      <p class="api-card__content-description">
+        {{ content }}
+      </p>
+      <button class="api-card__content-button" i18n="@@logIn" mat-stroked-button color="primary">Learn more</button>
```

Comment:
same here --> make the i18n id specific to this button

Reply:
Done